### DISCUSSION TITLE
Replace Evince with Papers?

### DISCUSSION THREAD

**Discussion by ryonakano** [elementary/evince#85] [Ideas]
[Papers](https://gitlab.gnome.org/GNOME/Incubator/papers) is GTK4-ported version of Evince. It would be good to discuss if we ship Papers instead of Evince or not (maybe in elementary OS 9 or later).

**Comment by danirabbit** [comment]
Yeah I think I'm +1 on this, it probably needs a little granite patch but we can always mark our evince package as EOL and make papers replace it so I think everyone will get a smooth upgrade

**Comment by teamcons** [comment]
Ah i was going to suggest this.
Papers has a lot of the internals overhauled and modernized. From what i heard is more actively developed.
It would be really fitting, following the current moving Pantheon to more modern tech (Wayland and gang), to go with more modern versions/forks.

**Comment by teamcons** [comment]
Should i in the meantime build and maintain a "Papers - Granitified" repo ?
Maybe with something like this accessible in advance, we can let early adopter try and see what issues could arise from this.
Papers has much more Rust elements. Notably.

**Reply by danirabbit** [reply]
I don't think you need to make a whole separate repo 🤔 probably could just propose a branch into the existing evince one. It might be less work

**Comment by ryonakano** [comment]
Created #98 from this Discussion, so closing this in favor of it.
